lib/widgets/message_list.dart

```diff
@@ -30,9 +30,8 @@ class MessageListTheme extends ThemeExtension<MessageListTheme> {
   MessageListTheme.light() :
```

Comment:
commit message nit: start the summary message with an imperative verb:

```
theme: Added labelTime variable in MessageListTheme 
```
- (with other changes) ->
```
theme: Replace dateSeparatorText and messageTimestamp with labelTime
```